dev-util/perf/files/perf-6.6-ia64.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tools/arch/ia64/include/asm/barrier.h b/tools/arch/ia64/include/asm/barrier.h
```

Comment:
Can you include the error at the top so we can easily verify if it's still needed in future or something?